### DISCUSSION TITLE
how to disable app

### DISCUSSION THREAD

**Discussion by tomdavidson** [InfiniTimeOrg/InfiniTime#1725] [Q&A] [answered]
Hello, one of my kids is excessively playing games on his PineTime. How can I remove the games from the menu or disable them?

**Comment by FintasticMan** [comment, marked as answer]
You would need to edit the code to remove the app from the menu, the switch case in DisplayApp and the source files array in `src/CMakeLists.txt`. There is currently no way to do this at runtime, but it's a feature we want to implement.
